openresty/tests/test.bats

```diff
@@ -0,0 +1,16 @@
+TEST_PKG_VERSION="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
+
+@test "Version matches" {
+  result="$(hab pkg exec ${TEST_PKG_IDENT} openresty -v 2>&1 | head -n 1 | awk -F'/' '{print $2}')"
+  [ "$result" = "${TEST_PKG_VERSION}" ]
+}
+
+@test "Service is running" {
+  echo -e "$(hab svc status)"
```

Comment:
Is this a debug echo that was left in accidentally? If not, maybe we should say

echo "Service status is: $(hab svc status)"

to make it more clear that this is debug output.

One note that I've seen, is that it appears that bats eats this output when doing `hab studio run`, so I'm not sure if this is useful either.

Reply:
Fixed.
Bats doesn't eat debug output if you enable TAPS. `CI=1`

Reply:
Ooooooo that's awesome <3